spec/lib/rdstation/retryable_request_spec.rb

```diff
@@ -74,6 +74,24 @@ class DummyClass
           end.not_to raise_error
         end
 
+        it 'refreshes the access_token and retries the request an Unauthorized error happen' do
```

Comment:
I guess you forgot the word `when`

```suggestion
        it 'refreshes the access_token and retries the request an Unauthorized error happen' do
```